static/app/views/insights/common/components/fullSpanDescription.tsx

```diff
@@ -53,24 +76,38 @@ export function FullSpanDescription({
   if (moduleName === ModuleName.DB) {
     if (system === 'mongodb') {
       let stringifiedQuery = '';
+      let result: ReturnType<typeof prettyPrintJsonString> | undefined = undefined;
 
-      if (fullSpan?.sentry_tags) {
-        stringifiedQuery = prettyPrintJsonString(fullSpan?.sentry_tags?.description);
+      if (indexedSpan?.['span.description']) {
+        result = prettyPrintJsonString(indexedSpan?.['span.description']);
       } else if (description) {
-        stringifiedQuery = prettyPrintJsonString(description);
+        result = prettyPrintJsonString(description);
       } else if (fullSpan?.sentry_tags?.description) {
-        stringifiedQuery = prettyPrintJsonString(fullSpan?.sentry_tags?.description);
+        result = prettyPrintJsonString(fullSpan?.sentry_tags.description);
+      } else if (fullSpan?.sentry_tags?.description) {
+        result = prettyPrintJsonString(fullSpan?.sentry_tags?.description);
       } else {
         stringifiedQuery = description || fullSpan?.sentry_tags?.description || 'N/A';
       }
 
-      return <CodeSnippet language="json">{stringifiedQuery}</CodeSnippet>;
+      if (result) {
+        const {prettifiedQuery} = result;
+        stringifiedQuery = prettifiedQuery;
```

Comment:
we could just squeeze this in one line
```suggestion
        stringifiedQuery = result.prettifiedQuery;
```